embark-ui/src/components/Converter.js

```diff
@@ -27,8 +27,24 @@ class Converter extends React.Component {
   }
 
   handleOnChange(event, key) {
-    const newUnits = calculateUnits(event.target.value, key);
-    this.setState({etherConversions: newUnits});
+    const value = event.target.value;
+    let newUnits;
+    if (value.slice(-1) === '.') {
```

Comment:
If there is a dot, shouldn't we just not do anything for now? This would become simpler.

Reply:
That was my first take too, but we have to call `setState()` in order to reflect the change back in the view.

Reply:
Ok makes sense.